manim/utils/hashing.py

```diff
@@ -139,6 +212,11 @@ def get_hash_from_play_call(camera_object, animations_list, current_mobjects_lis
     :class:`str`
         A string concatenation of the respective hashes of `camera_object`, `animations_list` and `current_mobjects_list`, separated by `_`.
     """
+    logger.debug("Hashing ...")
```

Comment:
Do you think we could include a progressbar here instead of just this statement?

Reply:
I think it's a bad idea. It would mean we would have two progress bars for one animations which .. can quickly become messy. 

Btw, this is only printed in debug level mode, so it won't appear in most of the uses.

Reply:
Okay, then, yes, it's a bad idea. I think it is enough to just print the time taken, like we are doing now.